sdk/trace/tracetest/example_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package tracetest_test
+
+import (
+	"context"
+	"fmt"
+
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/sdk/trace/tracetest"
+)
+
+func Example_simulateWorkflow() {
+	ctx := context.Background()
+
+	// Set up an in-memory span recorder and tracer provider.
+	recorder := tracetest.NewSpanRecorder()
```

Comment:
```suggestion
	sr := tracetest.NewSpanRecorder()
```

this is how we usually name it: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-go-contrib+%22tracetest.NewSpanRecorder%22&type=code